students/jesse_miller/session09/cli_main.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+'''
+This is the command interface for the mailroom.  Now that I know that donor
+manipulations work correctly (At least according to pytest.  I mean, I wrote
+the tests so the likelihood of human error here is high, you know?).
+'''
+#import os
+import sys
+from donor_models import DonorCollection, DonorTools
+from mail_box import MailBox
+'''
+Module imports
+'''
+
+#pylint: disable=C0103
+name = ''
+alms = DonorCollection()
+tools = DonorTools(name)
```

Comment:
No need for this one.

Reply:
Yeah, that's a legacy from me experimenting with different ideas.  :\